package.json

```
@@ -2,7 +2,7 @@
   "name": "iotagent-node-lib",
   "license": "AGPL-3.0-only",
   "description": "IoT Agent library to interface with NGSI Context Broker",
-  "version": "3.4.0-next",
+  "version": "3.4.3-next",
```

```suggestion
  "version": "3.4.0-next",
```

As the semantic for "x.y.0-next" use to be "the next .0 versfion after x.y.0" (typically, 3.5.0)